src/directives/filereader.js

```diff
@@ -15,6 +15,8 @@ goog.require('ngeo');
  *      <input type="file" ngeo-filereader="ctrl.fileContent"
  *        ngeo-filereader-supported="ctrl.supported"/>
  *
+ * @htmlAttribute {string} ngeo-filereader The content of the file read
+ * @htmlAttribute {boolean} ngeo-filereader-supported Whether the FileReader API is supported
```

Comment:
> Perhaps we can even say something when the attribute is declared "one-time bound" (&) and when it's not (=) ?

For me not, because it's for the user, not the developers.